src/scripts/get-wpt-revision.py

```diff
@@ -64,6 +94,18 @@ def main(interval):
 
 if __name__ == '__main__':
     parser = argparse.ArgumentParser(description=main.__doc__)
+    parser.add_argument('--retry-interval',
+                        type=int,
+                        default=60,
+                        help='''Duration in seconds to wait between retrying
+                            failed requests to wpt.fyi'''
+                        )
+    parser.add_argument('--retry-timeout',
+                        type=int,
+                        default=60 * 1000,
```

Comment:
Default timeout is, 1000 minutes? The timeout specified in`master.cfg` (600 seconds / 10 minutes) seems more reasonable. Or is this default value intentional?

Reply:
Another typo :(